test/specs/commonTests/handlesAccessibility.tsx

```diff
@@ -1,33 +1,47 @@
 import * as React from 'react'
 
 import { getTestingRenderedComponent } from 'test/utils'
-import { ButtonBehavior, DefaultBehavior } from 'src/lib/accessibility'
+import { DefaultBehavior } from 'src/lib/accessibility'
+import { Accessibility, AriaRole } from 'src/lib/accessibility/interfaces'
+import { FocusZone, FOCUSZONE_ID_ATTRIBUTE } from 'src/lib/accessibility/FocusZone'
 
 export const getRenderedAttribute = (renderedComponent, propName, partSelector) => {
   const target = partSelector
     ? renderedComponent.render().find(partSelector)
     : renderedComponent.render()
 
-  return target.first().prop(propName)
+  let node = target.first()
+  if (node.attr(FOCUSZONE_ID_ATTRIBUTE)) {
+    node = node.children().first() // traverse through FocusZone <div>
+  }
+  return node.prop(propName)
 }
 
+const overriddenRootRole = 'test-mock-role' as AriaRole
+
+const TestBehavior: Accessibility = (props: any) => ({
+  attributes: {
+    root: {
+      role: overriddenRootRole,
+    },
+  },
+})
+
 /**
  * Assert Component handles accessibility attributes correctly.
  * @param {React.Component|Function} Component A component that should conform.
  * @param {Object} [options={}]
  * @param {Object} [options.requiredProps={}] Props required to render Component without errors or warnings.
  * @param {string} [options.defaultRootRole=''] Default root role rendered when no override provided
- * @param {string} [options.accessibilityOverride=ButtonBehavior] Override to test accessibility property override
- * @param {string} [options.overriddenRootRole=ButtonBehavior] Overridden root role when accessibility property overridden
  * @param {string} [options.partSelector=''] Selector to scope the test to a part
+ * @param {boolean} [options.wrappedInFocusZone=false] If the component should be wrapped in FocusZone
  */
 export default (Component, options: any = {}) => {
   const {
     requiredProps = {},
-    defaultRootRole = undefined,
-    accessibilityOverride = ButtonBehavior,
-    overriddenRootRole = 'button',
+    defaultRootRole,
     partSelector = '',
+    isWrappedInFocusZone = false,
```

Comment:
In the event we change focus implementation details, this option might not make sense.  Suggest something generic like `implementsFocusZone` which is isolated against wrapping or other implementation changes.

Reply:
Replaced with `focusZoneProps` so we won't need to change all unit tests if implementation changes.